src/connectionController.ts

```diff
@@ -202,10 +202,22 @@ export default class ConnectionController {
       )
     ).filter((conn): conn is FailedMigrationConnectionDescriptor => !!conn);
 
-    if (Object.keys(this._connections).length) {
+    const loadedConnections = Object.values(this._connections);
+    if (loadedConnections.length) {
       this.eventEmitter.emit(DataServiceEventTypes.CONNECTIONS_DID_CHANGE);
     }
 
+    this._telemetryService.track(TelemetryEventTypes.SAVED_CONNECTIONS_LOADED, {
+      totalConnections: loadedConnections.length,
+      connectionsWithSecretsInKeytar: loadedConnections.filter(
```

Comment:
FYI Maz usually names telemetry props like this `total_connections`. Maybe it would be nice to double-check with him. For KeytarSecretsMigrationFailedProperties as well.

Reply:
Thank you @alenakhineika. I will double check and make the required changes.

Reply:
Addressed with the changes done in [`6e8dd7f` (#565)](https://github.com/mongodb-js/vscode/pull/565/commits/6e8dd7f7595f514bedbeda811424780d8ea4a6f0)